### ISSUE TITLE
feat/Make apt available when basing images off of Unstructured Ubuntu

### ISSUE THREAD

**Issue by jonasheschl** [Unstructured-IO/unstructured#3814]
The build system for the unstructured Ubuntu image seems to remove `apt` from the image. This is an issue when basing a custom image off of the Unstructured Ubuntu image. In these cases, most consumers will want to install additional packages. With the current Unstructured Ubuntu image, this is not easily possible.

The following Dockerfile for instance, would error out:

```docker
FROM downloads.unstructured.io/unstructured-io/unstructured:latest

RUN apt-get install -y clang gcc

[...]
```

It would be very useful for `apt`/`apt-get` to remain available up to the last layer on the Docker image.

**Comment by fury0928** [comment]
Hi @badGarnet nice to meet you
May I work on this issue for contribution?
Thank you

**Comment by fury0928** [comment]
cc: @ryannikolaidis

**Comment by badGarnet** [comment]
> Hi [@badGarnet](https://github.com/badGarnet) nice to meet you May I work on this issue for contribution? Thank you

I don't believe the build system removes `apt` from the image. But it does switch user to a non-root `notebook-user`
So for the issue the solution should be in the new dockerfile first switch user to root
```
FROM downloads.unstructured.io/unstructured-io/unstructured:latest

USER root
RUN apt-get install -y clang gcc
USER notebook-user
```
